### PR TITLE
Rename Startness to Startup

### PR DESCRIPTION
**Pull request by xstefank**

### CODE REVIEW

**Comment by xstefank** [comment]
@pgunapal can you please check?

File: api/src/main/java/org/eclipse/microprofile/health/Startup.java [hunk 1]
```diff
@@ -45,15 +45,15 @@
 @Documented
 @Retention(RetentionPolicy.RUNTIME)
 @Target({TYPE, METHOD, PARAMETER, FIELD})
```
Comment (Emily-Jiang) [inline api/src/main/java/org/eclipse/microprofile/health/Startup.java:47]: I noticed this annotation can be used on class, method, parameter and field. The TCKs only assert on class level annotation. As per my comments on the other PR regarding add this target on Liveness and Readiness, I suggest some tests to be added to ensure the usage on method, parameter, field were honoured. Thoughts?

Comment (xstefank) [reply api/src/main/java/org/eclipse/microprofile/health/Startup.java:47]: @Emily-Jiang good idea. Can you please raise it as an issue? We shouldn't discuss that in this PR.

File: spec/src/main/asciidoc/java-api.asciidoc [hunk 2]
```diff
@@ -60,13 +60,13 @@ This means that if this procedure fails the application can be discarded (termin
...
-=== Startness check
+=== Startup check
 
-A Health check for startness allows applications to define startup probes that
+A Health check for startup allows applications to define startup probes that
 are used for initial verification of the application before the liveness probe takes over.
 This is useful for applications which require additional startup time on their first initialization.
 
-The `@Startness` annotation must be applied on a `HealthCheck` implementation to define a startness check procedure, otherwise, this annotation is ignored.
+The `Startup` annotation must be applied on a `HealthCheck` implementation to define a startup check procedure, otherwise, this annotation is ignored.
```
Comment (pgunapal) [inline spec/src/main/asciidoc/java-api.asciidoc:69]: Shouldn't this be "The `@Startup` annotation..."

File: spec/src/main/asciidoc/java-api.asciidoc [hunk 3]
```diff
@@ -150,7 +150,7 @@ public class CheckDiskspace implements HealthCheck {
 
 == Integration with CDI
 
-Any enabled bean with a bean of type `org.eclipse.microprofile.health.HealthCheck` and `@Liveness`, `@Readiness`, or `@Startness` qualifier can be used as health check procedure.
+Any enabled bean with a bean of type `org.eclipse.microprofile.health.HealthCheck` and `@Liveness`, `@Readiness`, or `Startup` qualifier can be used as health check procedure.
```
Comment (pgunapal) [inline spec/src/main/asciidoc/java-api.asciidoc:153]: As well as here, it should be `@Startup`

File: spec/src/main/asciidoc/overview.asciidoc [hunk 4]
```diff
@@ -27,8 +27,8 @@ https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-read
 
 In this scenario health checks are used to determine if a computing node needs to be discarded (terminated, shutdown) and eventually replaced by another (healthy) instance.
 
-The MicroProfile Health architecture consists of three `/health/ready`, `/health/live` and `/health/start` endpoints in a MicroProfile runtime that respectively represent the readiness, the liveness and the startness of the entire runtime.
-These endpoints are linked to health check procedures defined with specifications API and annotated respectively with `@Readiness`, `@Liveness` and `@Startness` annotations.
+The MicroProfile Health architecture consists of three `/health/ready`, `/health/live` and `/health/start` endpoints in a MicroProfile runtime that respectively represent the readiness, the liveness and the startup health of the entire runtime.
+These endpoints are linked to health check procedures defined with specifications API and annotated respectively with `@Readiness`, `@Liveness` and `Startup` annotations.
```
Comment (pgunapal) [inline spec/src/main/asciidoc/overview.asciidoc:31]: Should be `@Startup`

**Review by pgunapal — CHANGES_REQUESTED** [review]
Looks good, minor typo fixes.

**Comment by xstefank** [comment]
@pgunapal thanks, fixed, can you check once more please?

File: spec/src/main/asciidoc/java-api.asciidoc [hunk 5]
```diff
@@ -42,7 +42,7 @@ The nature of the procedure is defined by annotating the `HealthCheck` procedure
 
 * Readiness checks defined with `@Readiness` annotation
 * Liveness checks defined with `@Liveness` annotation
-* Startness checks defined with `@Startness` annotation
+* Startup checks defined with `Startup` annotation
```
Comment (pgunapal) [inline spec/src/main/asciidoc/java-api.asciidoc:45]: Should be `@Startup` annotation

**Review by pgunapal — CHANGES_REQUESTED** [review]
Sorry, missed one more minor typo

**Review by pgunapal — APPROVED** [review]
LGTM

**Pull request merged**